packages/dmn-js-shared/src/features/expression-languages/ExpressionLanguages.js

```diff
@@ -0,0 +1,133 @@
+import { assign } from 'min-dash';
+
+
+const EXPRESSION_LANGUAGE_OPTIONS = [{
+  label: 'FEEL',
+  value: 'feel'
+}, {
+  label: 'JUEL',
+  value: 'juel'
+}, {
+  label: 'JavaScript',
+  value: 'javascript'
+}, {
+  label: 'Groovy',
+  value: 'groovy'
+}, {
+  label: 'Python',
+  value: 'python'
+}, {
+  label: 'JRuby',
+  value: 'jruby'
+}];
+
+/**
+ * @typedef ExpressionLanguageDescriptor
+ * @property {string} value - value inserted into XML
+ * @property {string} label - human-readable label
+ */
+
+/**
+ * Provide options and defaults of expression languages via config.
+ *
+ * @example
+ *
+ * // there will be two languages available with FEEL as default
+ * const editor = new DmnJS({
+ *   expressionLanguages: {
+ *     options: [{
+ *       value: 'feel',
+ *       label: 'FEEL'
+ *     }, {
+ *       value: 'juel',
+ *       label: 'JUEL'
+ *     }],
+ *     defaults: {
+ *       editor: 'feel'
+ *     }
+ *   }
+ * })
+ */
+export default class ExpressionLanguages {
+  constructor(injector) {
+    this._injector = injector;
+
+    const config = injector.get('config.expressionLanguages') || {};
+
+    this._config = {
+      options: EXPRESSION_LANGUAGE_OPTIONS,
+      defaults: {
+        editor: 'juel',
```

Comment:
This indeed seems to be a breaking change: Previously the default expression language was `JUEL` in some places:

https://github.com/bpmn-io/dmn-js/pull/441/files#diff-7e4c370bbd7cff5b10aa86ee6d46b5f0L156
https://github.com/bpmn-io/dmn-js/pull/441/files#diff-39cd139761c02e1a63cafe3d4e248a49L11
https://github.com/bpmn-io/dmn-js/pull/441/files#diff-b6d2b22098e254991c1616651cb826c1L9

It was already `juel` in other places though:

https://github.com/bpmn-io/dmn-js/pull/441/files#diff-7e4c370bbd7cff5b10aa86ee6d46b5f0L137

Could you verify with the Camunda platform team if the capitalization matters at all when executing DMN? It looks like lower case is the way to go, at lease [looking at the DMN engine documentation](https://docs.camunda.org/manual/latest/reference/dmn11/decision-literal-expression/) and [scriptTask documentation](https://docs.camunda.org/manual/7.11/reference/bpmn20/tasks/script-task/). Or maybe the DMN standard documents can give hints whether it is `feel` or `FEEL`. The old behavior (mixing CAPITALIZED and lower cased languages) is clearly broken and deserves a cleanup. I'd just like to double check we're unifying into the right direction.

Reply:
I'll clarify this with the Camunda platform team. If I understand DMN 1.1 Spec correctly, this should actually be an URI (cf. [page 41 of the spec](https://www.omg.org/spec/DMN/1.1/PDF#page=41)) :D

Reply:
But for the Script Task it has to be [_mime-type format_](https://www.omg.org/spec/BPMN/2.0/PDF#page=195).

Reply:
Thanks [for investigating](https://github.com/bpmn-io/dmn-js/pull/441#issuecomment-561194710).